docs/services/activejob.md

```diff
@@ -0,0 +1,5 @@
+# ActiveJob
+
+For some kind of actions to work (for instance, user registration) you need to configure an [ActiveJob](http://edgeguides.rubyonrails.org/active_job_basics.html) backend on your application.
+
+If you don't want to have any other dependency, you can use [delayed_job](https://github.com/collectiveidea/delayed_job/), although Decidim is agnostic on which kind of backend do you use (ie sidekiq, resque, etc).
```

Comment:
I think I know what you mean here, but I think it could be explain better. Like:

"""
If you don't want to introduce any other external dependencies for `decidim` (such as `redis`), you can use the `delayed_job` backend. That setup only depends on an underlying database which is already a requirement for `decidim` to work. Add the [delayed_job](https://github.com/collectiveidea/delayed_job/) gem to your `Gemfile` and follow the instructions in its readme to set it up.

However, Decidim is agnostic on which kind of backend do you use, so feel free the set up the backend (ie sidekiq, resque, etc) you like the most.
"""

Reply:
Yes, that's exactly what I mean, I think it's much cleaner. I've just updated my PR

Reply:
Thanks @andreslucena. As always, fantastic documentation PRs :muscle: